spec/features/user_views_all_landmarks_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+# [] Visiting the `/landmarks` path should contain a list of ordered landmarks.
+
+feature "user sees a list of landmarks" do
+  scenario "[success!] sees a list of landmarks in alphabetical order" do
+    statue_of_liberty = Landmark.create(
+    name: "Statue of Liberty",
+    location: "New York",
+    image: "https://upload.wikimedia.org/wikipedia/commons/thumb/a/a1/Statue_of_Liberty_7.jpg/500px-Statue_of_Liberty_7.jpg",
+    description: "A colossal neoclassical sculpture on Liberty Island"
+    )
+
+    grand_canyon = Landmark.create(
+    name: "Grand Canyon",
+    location: "Arizona",
+    image: "https://lh6.googleusercontent.com/-0IU2MaN1t0c/AAAAAAAAAAI/AAAAAAAAABs/ROeqMxHHRfY/s0-c-k-no-ns/photo.jpg",
```

Comment:
This should be fine, but you might want to take a look at image hosting sites (imgur, amazon S3) just in case this link is ever broken.